manager/src/main/java/com/redhat/service/bridge/manager/providers/WorkerIdProviderImpl.java

```diff
@@ -0,0 +1,16 @@
+package com.redhat.service.bridge.manager.providers;
+
+import java.util.UUID;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class WorkerIdProviderImpl implements WorkerIdProvider {
+
+    private static final String WORKER_ID = UUID.randomUUID().toString();
+
+    @Override
+    public String getWorkerId() {
+        return WORKER_ID;
```

Comment:
Since this class is `@ApplicationScoped` isn't this likely to return the same ID for all `ManagedResource`'?

Reply:
Yes, this is what the workerId is supposed to do

Reply:
The workerId identifies a application instance. Since the application can have multiple replica, every instance has its own workerId

Reply:
Ok. Understood. I thought it was meant to be per `ManagedResource`.